src/storage/BaseProcessor.h

```diff
@@ -89,28 +90,24 @@ class BaseProcessor {
                        const std::string& start,
                        const std::string& end);
 
-    cpp2::ErrorCode to(kvstore::ResultCode code);
-
-    cpp2::ErrorCode writeResultTo(WriteResult code, bool isEdge);
+    nebula::cpp2::ErrorCode writeResultTo(WriteResult code, bool isEdge);
 
     nebula::meta::cpp2::ColumnDef columnDef(std::string name,
                                             nebula::meta::cpp2::PropertyType type);
 
-    void pushResultCode(cpp2::ErrorCode code, PartitionID partId);
+    void pushResultCode(nebula::cpp2::ErrorCode code, PartitionID partId);
 
-    void pushResultCode(cpp2::ErrorCode code, PartitionID partId, HostAddr leader);
+    void pushResultCode(nebula::cpp2::ErrorCode code, PartitionID partId, HostAddr leader);
```

Comment:
Check if we need this one with extra `HostAddr leader`, if not in used, could delete it as well.

Reply:
handleLeaderChanged  need this function. 
 I added a default value to the `leader`, and then removed `pushResultCode `without` HostAddr leader`